### DISCUSSION TITLE
Question: How to remove empty commit from git-svn?

### DISCUSSION THREAD

**Discussion by NickLiu635** [git-for-windows/git#5133] [Q&A]
I use `--ignore-paths` to filter some files out when using `git svn init`, which makes git-svn generate some empty commit.
`> git log --oneline --stat`
![图片](https://github.com/user-attachments/assets/38aa8a71-130b-4dba-9448-15193595d374)
How to remove empty commit from git-svn?

**Comment by dscho** [comment]
Since this does not seem to be a Windows-specific question, you may instead want to take it to [the Git mailing list](mailto:git@vger.kernel.org) (send plain-text messages, HTML messages are dropped silently).
